HomeCafeRecipes/HomeCafeRecipes/Presentation/Feed/View/RecipeDetailViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  RecipeDetailViewController.swift
+//  HomeCafeRecipes
+//
+//  Created by 김건호 on 6/14/24.
+//
+
+import UIKit
+import RxSwift
+
+final class RecipeDetailViewController: UIViewController {
+    
+    private let recipeDetailView = RecipeDetailView()
```

Comment:
```suggestion
    private let contentView = RecipeDetailView()
```

뷰컨에서는 contentView로 들고있는게 좋아보여요

Reply:
[[60fa4e2](https://github.com/f-lab-edu/HomeCafeRecipes/pull/12/commits/60fa4e2f463198b1b2acf53f8b12d4e3da79b6ed)] 수정했습니다